RecoHI/HiJetAlgos/plugins/HiBadParticleCleaner.cc

```diff
@@ -91,7 +92,11 @@ void HiBadParticleCleaner::produce(edm::StreamID, edm::Event& iEvent, const edm:
 
   bool foundBadCandidate = false;
 
+  size_t n = pfCandidates->size();
+  std::vector<int> oldToNew(n);
+  size_t iPF = -1;
```

Comment:
`size_t` is an unsigned integral type: how can you initialize it to "-1"?
Probably `iPF++` does the right job nonetheless here below, but it is at least confusing using an unsigned type as such (and I think a few compilers could also complain)